lib/index.js

```diff
@@ -40,7 +40,9 @@ exports.register = (server, options, next) => {
   });
 
   server.ext('onPreResponse', (request, reply) => {
-    const rate = request.plugins['hapi-rate-limit'].rate;
+
+    /* istanbul ignore next */
```

Comment:
You said that this ternary was necessary because if post auth is skipped, this could err, right? If you have a scenario in mind that makes this needed, you should test it instead of ignoring it.